ee/codegen/src/generators/nodes/bases/base.ts

```diff
@@ -829,7 +832,19 @@ class NodeImplementationFile<
   }
 
   protected getFileStatements(): AstNode[] {
-    return [this.node.generateNodeClass()];
+    const statements: AstNode[] = [];
+    // This is to generate functions in the same file as inline prompt node if present
+    if (isInlinePromptNode(this.node.nodeData)) {
```

Comment:
[blocking] logic that is Inline Prompt Node-specific should live in the inline prompt node generator. We want to have this base file free of all _node_ level special casing, and instead, know how to generate _attributes_, in case other nodes have a `functions` attribute to generate.

So we should either:
- just move this to inline-prompt-node generator
- add handling to our Node Attributes logic